src/index.spec.ts

```diff
@@ -21,9 +22,10 @@ describe('EppoJSClient E2E test', () => {
   let globalClient: IEppoClient;
 
   const flagKey = 'mock-experiment';
+  const hashedFlagKey = md5(flagKey);
```

Comment:
A separate variable because we still pass in the un-hashed flag key to get assignments. We use the hashed version for mocking the config store